roles/maas_pacemaker/tasks/main.yaml

```diff
@@ -0,0 +1,39 @@
+---
+- name: "Install Pacemaker packages"
+  ansible.builtin.apt:
+    name: "{{ item }}"
+    state: present
+    update-cache: true
+  with_items:
+    - pacemaker
+    - pcs
+    - fence-agents
+
+- name: "Ensure Pacemaker is started"
+  ansible.builtin.systemd:
+    name: "pacemaker.service"
+    state: started
+    enabled: false # if the system is rebooted, the user may not want this member to rejoin automatically
+
+- name: "Check if fencing is configured"
+  ansible.builtin.shell:
+    cmd: "set -opipefail; pcs stonith status | tee {{ maas_pacemaker_tmp_status }}"
```

Comment:
Should this be 'set -o pipefail' instead? I can't get pcs stonith status to work, should this be a different command (pcs stonith config, or something else)? I also need to change the shell to bin/bash (adding args: executable: bin/bash) because set -o pipefail does not seem to be a valid command in bin/sh?